Because Poky doesn't support at_console we need to
special-case the session user.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Jussi Kukkonen <jussi.kukkonen@intel.com>

---
 src/connman-dbus.conf | 3 +++
 vpn/vpn-dbus.conf     | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/src/connman-dbus.conf b/src/connman-dbus.conf
index 98a773e..466809c 100644
--- a/src/connman-dbus.conf
+++ b/src/connman-dbus.conf
@@ -8,6 +8,9 @@
         <allow send_interface="net.connman.Counter"/>
         <allow send_interface="net.connman.Notification"/>
     </policy>
+    <policy user="xuser">
+        <allow send_destination="net.connman"/>
+    </policy>
     <policy at_console="true">
         <allow send_destination="net.connman"/>
     </policy>
diff --git a/vpn/vpn-dbus.conf b/vpn/vpn-dbus.conf
index 0f0c8da..9ad05b9 100644
--- a/vpn/vpn-dbus.conf
+++ b/vpn/vpn-dbus.conf
@@ -6,6 +6,9 @@
         <allow send_destination="net.connman.vpn"/>
         <allow send_interface="net.connman.vpn.Agent"/>
     </policy>
+    <policy user="xuser">
+        <allow send_destination="net.connman.vpn"/>
+    </policy>
     <policy at_console="true">
         <allow send_destination="net.connman.vpn"/>
     </policy>
-- 
2.1.4

